pkg/tfbridge/walk_test.go

```diff
@@ -95,12 +102,18 @@ func TestPropertyPathToSchemaPath(t *testing.T) {
 		},
 		{
 			name:     "list",
-			pp:       []any{"listStr"},
+			pp:       []any{"listStrs"},
```

Comment:
Was there a bug here? Why is the test changing?

Reply:
From the commit message of https://github.com/pulumi/pulumi-terraform-bridge/pull/1590/commits/eca5d902fabd438d6224e0e0c96d4dfeb4c13e2a:

```
Note:

Some tests had PP names that didn't make sense with the given schema. I corrected them to
allow round-tripability:

`` `patch
-			pp:       []any{"objSet", 0},
+			pp:       []any{"objSets", 0},
`` `

I added a test with name overrides to confirm that it still worked as expected.
```

Basically, the property path `[]any{"listStr"}` would have never been generated by the bridge from it's schema, since `TerraformToPulumiName` would have pluralized `listStr` to `listStrs`. This worked in the context of the previous test, but lost information in the translation.

---

P.S. I really don't like that GH doesn't surface commit messages.

Reply:
Ah sorry I'll make a point of reading through those next time. Glad there was not a prod bug.